src/components/SecondaryPanes/index.js

```diff
@@ -107,7 +108,28 @@ class SecondaryPanes extends Component<Props> {
         : L10N.getStr("breakpoints.disable")
     };
 
-    return <input {...inputProps} />;
+    if (!features.breakpointsDropdown) {
+      return;
```

Comment:
if the feature is not enabled we still want ` <input {...inputProps} />;` right?

Reply:
Yes good catch. I must have forgot to put that back.